view/components/features/login-form.tsx

```diff
@@ -26,6 +26,35 @@ export interface LoginFormProps {
 export function LoginForm({ ...props }: LoginFormProps) {
   const { t } = useTranslation();
   const [showPassword, setShowPassword] = useState(false);
+  const [emailError,setEmailError] = useState('');
+  const [passwordError,setPasswordError] = useState('');
+
+  function isValidEmail(email:string){
+  return /^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(email);
```

Comment:
can you add the split of this regex so whoever reads this get an idea of what each part does, or even for simplicity can you add a comment on top to explain how it works,